src/PotentialTypes/EmpiricalPotentials/empirical_potentials.jl

```diff
@@ -31,7 +30,7 @@ function energy_and_force(A::AbstractSystem, p::EmpiricalPotential)
             end
         end
     end
-    (; e, f)
+    (; e=e * u"hartree", f=f * u"hartree / bohr")
```

Comment:
I am not familiar with the "hartree" unit but I think the typical unit that practioners use is the "eV" and "eV / Angstrom"

Reply:
Yeah so the atomic unit standard is bohrs for distance and hartree for energy. This is the same standard used across UnitfulAtomic, Atomistic, and DFTK. I have no domain knowledge, but from what Michael has told me there is no single convention that everyone follows. From my perspective, this is actually the biggest reasons to use unitful: different people who have different (often implicit) unit conventions can both write software that works together by automatically handling the unit conversions and makes the convention chosen explicit on the output.

Here are some references:
- [UnitfulAtomic.jl](https://github.com/sostock/UnitfulAtomic.jl)
- [Wikipedia](https://en.wikipedia.org/wiki/Hartree_atomic_units)